rmgpy/chemkin.py

```diff
@@ -621,6 +621,23 @@ def readReactionComments(reaction, comments, read = True):
                 reaction.pairs.append((reactant, product))
             assert len(reaction.pairs) == max(len(reaction.reactants), len(reaction.products))
 
+        elif isinstance(reaction,TemplateReaction) and 'rate rule ' in line:
+            bracketed_rule = line.split('rate rule ')[-1]
+            templates = rule = bracketed_rule[1:-1].split(';')
+            reaction.template = templates
+            # still add kinetic comment
+            reaction.kinetics.comment += line.strip() + "\n"
+
+        elif isinstance(reaction,TemplateReaction) and\
+                       'Multiplied by reaction path degeneracy ' in line:
+            degen = float(line.split('Multiplied by reaction path degeneracy ')[-1])
+            reaction.degeneracy = degen
+            # undo the kinetic manipulation caused by setting degneracy
+            if reaction.kinetics:
+                reaction.kinetics.changeRate(1./degen)
```

Comment:
One way to avoid it would be to remake the kinetics object or have the kinetics object made all in one statement. The method of reading and executing line by line prevents this from being done in a clean way. That is also why I wrote a test to check the kinetics.